test/extended/builds/pipeline.go

```diff
@@ -13,36 +16,98 @@ import (
 	"github.com/openshift/origin/test/extended/util/jenkins"
 )
 
+func debugAnyJenkinsFailure(br *exutil.BuildResult, name string, oc *exutil.CLI, dumpMaster bool) {
+	if !br.BuildSuccess {
```

Comment:
I guess it's fine to leave it, but if we end up wanting to reuse this for debugging other failures, we're going to end up refactoring it.